blockchain/indexers/utreexoproofindex.go

```diff
@@ -236,6 +238,58 @@ func (idx *UtreexoProofIndex) Create(dbTx database.Tx) error {
 	return nil
 }
 
+// storeFilter stores a given filter, and performs the steps needed to
+// generate the filter's header.
+func storeUtreexoCFilter(dbTx database.Tx, block *btcutil.Block, filterData []byte,
```

Comment:
We don't need to store the data since that's already stored. We just need to store the filter header.